ef_tests/Cargo.toml

```diff
@@ -4,7 +4,8 @@ version.workspace = true
 edition.workspace = true
 
 [dependencies]
-revm = { version = "9.0.0", features = ["serde", "std", "serde-json"] }
 ethrex-core.workspace = true
 serde.workspace = true
 serde_json.workspace = true
+bytes.workspace = true
+hex = "0.4.3"
```

Comment:
since the `hex` dependency is also used in `core` we can move it to be a workspace level dep.

Reply:
I don't see it as a dependency in core's toml